docs/browser.md

```diff
@@ -0,0 +1,54 @@
+---
+id: browser
+title: Browser
+---
+
+Run Prettier in the browser with the `standalone.js` UMD bundle shipped in the NPM package (starting in version 1.13). The new UMD bundle only formats the code and has no support for config files, ignore files, CLI usage, or automatic loading of plugins.
```

Comment:
I just realized this, but we might also want to mention that `formatWithCursor` isn't present (unless that's unintended?)

Reply:
Probably an oversight on my side.. there’s no reason to leave it out!

Reply:
https://github.com/prettier/prettier/pull/4556